src/config/Config.zig

```diff
@@ -4866,6 +4866,13 @@ pub const Keybinds = struct {
                 .{ .jump_to_prompt = 1 },
             );
 
+            // Toggle command palette, matches VSCode
+            try self.set.put(
+                alloc,
+                .{ .key = .{ .translated = .p }, .mods = .{ .super = true, .shift = true } },
```

Comment:
Super+Shift+P sounds *very* wrong to me. AFAICT GNOME Builder uses Ctrl+Return, and VS Code on Linux uses Ctrl+Shift+P

Reply:
This only applies to macOS. The binding is unset on Linux. 

When we support GTK we can use our helper we already use all over to swap Ctrl for Super. Mac doesn’t use control for most shortcuts because it has the “command” (super) key. You can see that across most shared keybinds and popular programs like Chrome, Firefox, and even in your example VSCode.

Reply:
I mean I feel like we can do "correctly" (e.g. using `ctrlOrSuper`) in this PR, and just make the action be a no-op on GTK for now